provisioning/miniprov/hare_mp/store.py

```diff
@@ -41,3 +42,26 @@ def __init__(self, url: str):
 
     def _raw_get(self, key: str) -> str:
         return self.conf.get('hare', key)
+
+    def get_machine_id(self) -> str:
+        with open('/etc/machine_id', 'r') as f:
+            machine_id = f.readline().strip('\n')
+            hostname = self.conf.get('hare',
+                                     f'server_node>{machine_id}>hostname')
+            if hostname != socket.gethostname():
+                raise RuntimeError(f'invalid machine id {machine_id}')
+            return machine_id
+        raise RuntimeError(f'cannot read machine_id')
+
+    def get_cluster_id(self) -> str:
+        machine_id = self.get_machine_id()
+        cluster_id = self.conf.get('hare',
```

Comment:
The same thing. Let's replace self.conf.get with our `self._get_raw()`. It is more practical to have just one place that interacts with ConfStore.

Reply:
Done.